helm_deploy/whereabouts-api/values.yaml

```diff
@@ -36,6 +36,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     HMPPS_SQS_USE_WEB_TOKEN: "true"
+    FEATURE_BVLS_ENABLED: "true"
```

Comment:
When the time comes this will be set to 'false' and a deployment will be needed.